src/ortolan.ts

```diff
@@ -0,0 +1,57 @@
+import * as shell from '@actions/exec'
```

Comment:
Simplify import by using named import instead of wildcard.
```suggestion
import {exec) from '@actions/exec'
```
When suggestion is commited the `shell.exec` occurences should be replaced with `exec`.

Reply:
done in upcoming commit